tools/clang/test/CodeGenSPIRV/var.init.hlsl

```diff
@@ -52,6 +58,13 @@ float4 main(float component: COLOR) : SV_TARGET {
 // CHECK-NEXT: OpStore %o [[oinit]]
     float4 o = float4(1.0, j, 3.0, j);      // Mixed case
 
+    int2 p = {1, 2}; // All components are constants
```

Comment:
At line 29 in the above. Initializer is used for this case instead of `OpStore`.